pkg/components/exporter/exporter.go

```diff
@@ -148,10 +149,9 @@ func NewDaemonSet(k *v1alpha1.Kepler) *appsv1.DaemonSet {
 							{Name: "NODE_IP", ValueFrom: k8s.EnvFromField("status.hostIP")},
 							{Name: "NODE_NAME", ValueFrom: k8s.EnvFromField("spec.nodeName")}},
 						VolumeMounts: []corev1.VolumeMount{
-							{Name: "lib-modules", MountPath: "/lib/modules"},
-							{Name: "tracing", MountPath: "/sys"},
-							{Name: "kernel-src", MountPath: "/usr/src/kernels"},
-							{Name: "kernel-debug", MountPath: "/sys/kernel/debug"},
```

Comment:
If I understand correctly, it should be obsolete now that https://github.com/sustainable-computing-io/kepler/pull/752 is merged and released

Reply:
@BGrasnick Could you please clarify which release contains that PR? 

```
git tag --contains 885c1bc37ba18e00725149cf06c3f53c86846b59
```
returns nothing 🤔

Reply:
Also note that this version of operator deploy 0.5.4 version of kepler. See https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/pkg/components/exporter/exporter.go#L51 

Would you like to update that as well?

Reply:
I don't know what happened with the commit and its SHA but apparently the change is in the main branch and also releases 0.5.2, 0.5.3 and 0.5.4. It was first [merged](https://github.com/sustainable-computing-io/kepler/commit/4c9c150e4bff4edb854be6f9647c2b963f69b339), then [reverted ](https://github.com/sustainable-computing-io/kepler/commit/eba46bce1d0b1a9d0f2d466d18b4513a2aea767a )and the [revert was reverted](https://github.com/sustainable-computing-io/kepler/commit/9961baca1fd2e65fb2dd5cf6a173dafc91654d1f) again... 

But [in quay](https://quay.io/repository/sustainable_computing_io/kepler?tab=tags) there is also a release-0.5.3-libbpf, release-0.5.4-libbpf and latest-libbpf build.

Regarding the version I don't quite understand because as far as I can tell, this PR also uses release-0.5.4 version of kepler.

Also I just tried locally with our 4.11 OpenShift cluster and from what I can see this PR works with both works with latest and latest-libbpf images of kepler. There were some missing metrics when using release-0.5.4 but I think this is due to https://github.com/sustainable-computing-io/kepler/issues/877. So it should work with both bcc and libbpf without mounting sys/kernel/debug but I am no expert in that area and would love if you could confirm this. 

If I understand correctly, right now the user can choose whether they want to use bcc or libbpf by using a specific image of kepler. If sys/kernel/debug is necessary for bcc maybe we can make this optional and enable choosing bcc vs libbpf image in the spec of kepler CR and then optionally mount sys/kernel/debug? Is the plan for the future to offer both or go for only one of them?

Reply:
I am not sure about this either since it is @vimalk78 who has been looking into kepler implementation (including bcc) 

@rootfs , could you please help us resolve this? 
I guess the question is if we require `{Name: "kernel-debug", MountPath: "/sys/kernel/debug"},` mount for 0.5.4 to work properly given it uses bcc?

Reply:
@sthaha This is a good question. The k8s kepler exporter yaml doesn't have it now https://github.com/sustainable-computing-io/kepler/blob/main/manifests/config/exporter/exporter.yaml, but for some unknown reason it is required for openshift when first experimented https://github.com/sustainable-computing-io/kepler/blob/main/manifests/config/exporter/patch/patch-openshift.yaml

Would you please check if this is still the requirement for openshift? It would be great if kepler doesn't need this mount any more.

Reply:
@BGrasnick I would love to include this patch in the release we are going to make today. Since we don't have a consensus on whether `{Name: "kernel-debug", MountPath: "/sys/kernel/debug"}` is required, would you mind restoring that back which gives buys us time to properly investigate this issue?

Reply:
Yeah we could do that and separate it into another PR and then both together would fix the issue. If you don't mind, what is the release cadence so when could this /sys/kernel/debug fix be released in the future?